testing/oak_echo_raw_bin/README.md

```diff
@@ -1,4 +1,8 @@
-# Echo Baremetal
+# Oak Echo Raw
 
 Echo baremetal binary that can be run under
 [crosvm](https://chromium.googlesource.com/chromiumos/platform/crosvm/).
+
+This binary doesn't use IDL to communicate between the Untrusted Launcher and
+the guest VM, but instead uses a raw SimpleIO channel. The binary is used
+primarily for Vanadium tests.
```

Comment:
But what does it actually do? I would expect something like: it receives a message as bytes, and sends back the same bytes, without interpreting them.

Reply:
Done